plugins/labhub.py

```diff
@@ -65,7 +68,7 @@ def __init__(self, bot, name=None):
                              filter(lambda x: (x.full_name.split('/')[0] ==
                                                self.GH_ORG_NAME),
                                     self.IGH.write_repositories)}
-        except RuntimeError:
+        except RuntimeError:  # pragma: no cover, for logging
```

Comment:
you should be able to use requests-mock to break IGitt so it gives a RuntimeError , but ... not critical ... this is OK for now